modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/PrioritiesAccessor.java

```diff
@@ -52,6 +53,15 @@
     void updatePriorities(List<UUID> experimentPriorityList, String applicationName);
 
     /**
+     * Append experiments to the priorities
+     *
+     * @param experimentIds and experiment ids which is to be appended to the existing priorities
```

Comment:
Nit: Did you mean, "@param experimentIds experiment ids which are to be appended to the existing priorities"